packages/@react-spectrum/searchwithin/chromatic/SearchWithin.chromatic.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import {Item, Picker} from '@react-spectrum/picker';
+import React from 'react';
+import {SearchField} from '@react-spectrum/searchfield';
+import {SearchWithin, SpectrumSearchWithinProps} from '../';
+
+import {storiesOf} from '@storybook/react';
+
+storiesOf('SearchWithin', module)
```

Comment:
Just as a note, should probably convert this to the [CSF format ](https://storybook.js.org/docs/react/api/csf). I should've updated this in plop, my b. There are a couple of other chromatic stories here and there that you can use as a base (Tabs, ComboBox, etc). 

Maybe won't have to do the whole powerset generation that ComboBox/Color components/etc do, but if you are interested you can take a look at those chromatic stories as well

Reply:
actually, this is a good point for any stories we right, chromatic or not